spec.bs

```diff
@@ -134,6 +230,35 @@ Examples of [=task source/freezable=] task sources are:
 * networking: tasks queued due to loading of resources
 * DOM manipulation, user interaction, history traversal etc.
 
+<table><tbody>
```

Comment:
I somehow thing this enumeration might still be useful. I was originally tempted to add a 3rd column that indicated the "throttling" nature of these queues because they differ. But I didn't want to get into the definition of throttling.

Reply:
I agree it's nice to know all the task sources on the platform. We may want to consider moving this to HTML as a sort of registry, or similar. But I think the concept of a non-freezable task source just doesn't make sense in spec-land, and having it is confusing.

Reply:
How do we wish to handle the network task source running for a little bit?

Reply:
I guess I'd need more detail on what the implementation plan is there. Something like, freeze all networking tasks except any that are associated with a currently-ongoing request at the time of freezing?